packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1605,8 +1605,8 @@ abstract class RenderObjectElement<T extends RenderObjectWidget> extends Buildab
     while ((oldChildrenTop <= oldChildrenBottom) && (newChildrenTop <= newChildrenBottom)) {
```

Comment:
This function has other customers; I guess they need to handle detachChild also in due course.